mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -86,13 +90,24 @@ func TestClientSideEncryptionProse(t *testing.T) {
 			}},
 		}
 		schemaMap := map[string]interface{}{"db.coll": schema}
+		tlsMap := make(map[string]map[string]interface{})
+		tlsOptsMap := map[string]interface{}{
+			"tlsCertificateKeyFile": sslClientCertificateKeyFile,
+			"tlsCAFile":             sslCertificateAuthorityFile,
+		}
+		tlsMap["kmip"] = tlsOptsMap
+
 		aeo := options.AutoEncryption().
 			SetKmsProviders(fullKmsProvidersMap).
 			SetKeyVaultNamespace(kvNamespace).
 			SetSchemaMap(schemaMap)
+		_, err := aeo.SetTLSOptions(tlsMap)
+		assert.Nil(mt, err, "expected no error, got error: %v", err)
```

Comment:
```suggestion
		assert.Nil(mt, err, "SetTLSOptions error: %v", err)
```
Here and in other error messages that say "expected no error". Always easier to debug when we know where the error is coming from.

Reply:
Got it that seems a lot more helpful.